Modules/_sqlite/cache.c

```diff
@@ -113,7 +117,8 @@ void pysqlite_cache_dealloc(pysqlite_Cache* self)
     Py_DECREF(tp);
 }
 
-PyObject* pysqlite_cache_get(pysqlite_Cache* self, PyObject* key)
+extern PyObject *
```

Comment:
Is not `extern` redundant?

Reply:
Yes, but I like the explicity. I'm happy to remove it, though.

Reply:
Explicit `extern` is not commonly use in CPython sources, so it would be better to remove it.

Reply:
All right, I'll do that. Thanks.

Reply:
Fixed in bd5c34559f7b85328e0f90a67888d8dc1e740b1a. PTAL.